todolist/formatter.go

```diff
@@ -40,7 +40,7 @@ func (f *Formatter) Print(printNotes bool) {
 			f.printTodo(todo)
 			if printNotes {
 				for nid, note := range todo.Notes {
-					fmt.Fprintf(f.Writer, "   %s\t%s\t\n",
+					fmt.Fprintf(f.Writer, "   %s\t\t\t%s\t\n",
```

Comment:
would be nice to see a test when #109 is merged.

Reply:
sure.